package.json

```diff
@@ -53,7 +53,7 @@
   },
   "main": "./pencilblue.js",
   "engines": {
-    "node": ">= 0.11.0"
+    "node": ">= 0.11.0 <5"
```

Comment:
FYI... a unit test broke with node v5.  Until we can investigate further I've updated the docs and the package.json file to ensure that users are running with proper versions.